metis-enrichment/metis-enrichment-client/pom.xml

```diff
@@ -138,11 +129,7 @@
                 </exclusion>
             </exclusions>
         </dependency>
-		<dependency>
-		    <groupId>org.slf4j</groupId>
-		    <artifactId>slf4j-api</artifactId>
-		    <version>${version.slf4j}</version>
-		</dependency>
+	
```

Comment:
We shouldn't be moving to log4j, so this dependency should stay.

Reply:
Reverted.